frodo.js

```diff
@@ -2,35 +2,38 @@
  * Extras Plugin
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.